handlers/api.go

```diff
@@ -2292,15 +2315,15 @@ func ApiGraffitiwall(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	}
-	slotQuery = math.MaxU64(slotQuery, 10000)
+	slotQuery = utilMath.MaxU64(slotQuery, 10000)
```

Comment:
Why the switch to `utilMath`? This now is only used within this file. Searching `MinU64` at least shows one other `math` use:
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/37c6603b943dd149abacf0bf10a3d82ae1a950a6/handlers/validator.go#L1947

Reply:
I wanted to use the normal math dependency just for a single call of `math.MaxInt` and since 2 dependencies can't have the same name I had to rename one.

Reply:
My bad, I misread that.